src/include/internal_libreport.h

```diff
@@ -1148,6 +1148,7 @@ struct dump_dir *open_directory_for_writing(
 #define FILENAME_PKG_VERSION   "pkg_version"
 #define FILENAME_PKG_RELEASE   "pkg_release"
 #define FILENAME_PKG_ARCH      "pkg_arch"
+#define FILENAME_UNPACKAGED    "unpackaged"
```

Comment:
Why do you need the new file? What about to call a report "unpackaged" if the file `package` does not exist?